video.md

```diff
@@ -2,26 +2,19 @@
 
 # CodeRefinery: welcome and practical information for workshops
 
-## Arrival
-
-Please test the following when arriving:
-
-- Test that you have Git installed: `git --version`
-- Check [Git configuration instructions](https://coderefinery.github.io/git-refresher/01-setup/#configuring-git).
-
```

Comment:
Unrelated to this, but I want to move this to another file in this repo.  Having some standard info to project as people are walking in / connecting to the meeting would be useful.

Even if not used in every workshop, it's helpful to me to not have to think it up every time.

Reply:
I can leave it also in, just the way we open links we never spend more than 3 seconds on this so we need a better place. Should I spin off an issue or move this to a file? But will we then remember that the file exists?

Reply:
I think the right place for this in video workshops is the HackMD. So if we have a template, I think this is where it should be but here in the intro session it's IMO not well placed.

Reply:
Good point.  Really, same for in-person... this should become hackmd templates.